pkg/client/client.go

```diff
@@ -3,11 +3,12 @@ package client
 
 import (
 	"context"
-	"fmt"
 	"math/big"
 	"sync"
 	"time"
 
+	"github.com/keep-network/keep-common/pkg/chain/ethereum/ethutil"
```

Comment:
Oh, I see my suggestion about leaving confirmations in `ethutil` for now does not play well here. This code should not import anything ethereum-specific. True, it's already importing `go-ethereum/common` for `common.Address` but I don't think it's right. Let me fix it.

Reply:
Please see https://github.com/keep-network/keep-common/pull/57

Reply:
We still need to bump `keep-common` dependency and update calls.

Reply:
Done.